src/particle.cpp

```diff
@@ -47,67 +58,47 @@ void sample_particle::print_filename()
   return;
 }
 
-//read file
+//read file and push events to stack
```

Comment:
more descriptive comment, like reads the file in chunks of size size and pushes the events to the stack, etc